### ISSUE TITLE
Specifying variables doesn't seem to work

### ISSUE THREAD

**Issue by makenai** [adius/nodescad#3]
Hi, I tried to specify variables in the options array:

``` javascript
var options = {
  binaryPath: openSCADPath,
  inputFile: __dirname + '/scad/sumobot.scad',
  variables: {
    'cat': 'dog'
  }
}
```

However, the schema validator comes back with `Unknown property (not in schema)`. Taking out variables or passing an empty object seems to work fine.

**Comment by makenai** [comment]
Proposed fix in #4

**Comment by adius** [comment]
Fixed by c011e5f
